docs/basics.md

```diff
@@ -355,7 +355,7 @@ requests periodically carried out by Traefik. The check is defined by a path
 appended to the backend URL and an interval (given in a format understood by [time.ParseDuration](https://golang.org/pkg/time/#ParseDuration)) specifying how
 often the health check should be executed (the default being 30 seconds).
 Each backend must respond to the health check within 5 seconds.
-By default, the port of the backend server is used, however, this may be overridden.  
```

Comment:
same here

Reply:
There is no following lines after this line - this is the end of the paragraph. So I think it's okay to remove two spaces here.